Blockzilla/SettingsViewController.swift

```diff
@@ -100,7 +100,7 @@ class SettingsViewController: UIViewController, UITableViewDataSource, UITableVi
             case .search: return 2
             case .siri: return 3
             case .integration: return 1
-            case .mozilla: return 2
```

Comment:
This should only be displayed for users who actually see home screen tips. So you should check here if we are before adding it to the section and adding to the initial toggles. I think this needs some more breaking down since not every user sees home screen tips.